src/openzeppelin/access/ownable/ownable_camel.cairo

```diff
@@ -0,0 +1,35 @@
+#[starknet::contract]
+mod OwnableCamel {
```

Comment:
I know is not ideal as well, but Instead of moving Camel interfaces to its own module, I've been changing them to CamelOnly interfaces. I think is easier to use this temporal solution until we have components, instead of creating separated Camel modules. Wdyt? You can see an example in the ERC721 PR.

Reply:
Yeah, that solution is definitely easier. I'll update